bears/c_languages/CPPCheckBear.py

```diff
@@ -0,0 +1,35 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.settings.Setting import typed_list
+
+
+@linter(executable="cppcheck",
+        use_stdout=False,
```

Comment:
is this needed?

just doing `use_stderr=True` would work?

if there is any kind of useless output on stdout, you may ignore this.